src/core/server/audit_trail/types.ts

```diff
@@ -19,17 +19,78 @@
 import { KibanaRequest } from '../http';
 
 /**
- * Event to audit.
+ * Audit event using ECS format: https://www.elastic.co/guide/en/ecs/current/index.html
  * @public
  *
  * @remarks
  * Not a complete interface.
  */
-export interface AuditableEvent {
+export interface AuditEvent {
   message: string;
-  type: string;
+  event: {
```

Comment:
Just did a quick check of this structure vs what we're using for the alerting event log - looks good!  \o/ 

One thing to note is that we add `start`, `end` and `duration` (in nanoseconds!) in `event` for events where there is a duration that we can actually measure.  Seems like that _might_ be the case here, not sure.  It's probably not relevant for an audit log anyway, would be useful for other logging purposes - just thought I'd mention it.